app/helpers/application_helper.rb

```diff
@@ -22,4 +22,31 @@ def week(with_weekend = false)
       [:monday, :tuesday, :wednesday, :thursday, :friday]
     end
   end
+
+  def nationality_name(nationality)
```

Comment:
Everything that is only used by one model should be moved to the corresponding model. In helpers there should really just be helpers which do something with the view